src/AElf.Kernel.SmartContractExecution/Application/IBlockAttachService.cs

```diff
@@ -27,24 +28,48 @@ public class BlockAttachService : IBlockAttachService, ITransientDependency
             Logger = NullLogger<BlockAttachService>.Instance;
         }
 
+        public async Task AttachReceivedBlock(BlockWithTransactions blockWithTransactions)
+        {
+            Block block = blockWithTransactions.ToBlock();
+            
+            if (!await ValidateAsync(block))
+                return;
+            
+            await _blockchainService.AddBlockWithTransactionsAsync(blockWithTransactions);
+            await AttachAndExecute(block);
```

Comment:
Save txs in network while receive block from network

Reply:
you suggest saving from the BlockSyncJob ?